src/App.test.tsx

```diff
@@ -1,9 +1,16 @@
 import React from "react";
 import { render, screen } from "@testing-library/react";
-import App from "./App";
+import NumberGenerator from "./components/NumberGenerator";
```

Comment:
Number generator test cases should be in numberGenerator.test.tsx

Reply:
Resolved